public/javascripts/SVLabel/src/SVLabel/navigation/MapService.js

```diff
@@ -1450,9 +1451,9 @@ function MapService (canvas, neighborhoodModel, uiMap, params) {
      */
     function setViewControlLayerCursor(type) {
         switch(type) {
-            case 'ZoomOut':
+            /*case 'ZoomOut':
```

Comment:
@sagec108 You need not comment out the switch case. Correct way is not leave it uncommented and change the lines of code which uses this method.